packages/riipen-ui/src/components/TableContainer.jsx

```diff
@@ -0,0 +1,80 @@
+import clsx from "clsx";
+import PropTypes from "prop-types";
+import React from "react";
+
+import ThemeContext from "../styles/ThemeContext";
+
+class TableContainer extends React.Component {
+  static propTypes = {
+    /**
+     * Children to render in the table element
+     */
+    children: PropTypes.node,
+
+    /**
+     * Classes to apply to the root element
+     */
+    classes: PropTypes.arrayOf(PropTypes.string),
+
+    /**
+     * Whether to give all columns in the table equal width
+     */
+    fixed: PropTypes.bool,
+
+    /**
+     * Whether to center the table in the parent container
+     */
+    centered: PropTypes.bool,
+
+    /**
+     * Whether to render the background for a table
+     */
+    background: PropTypes.bool,
```

Comment:
Rename to `backgroundColor` and white list the available options (ie, "grey100", "transparent") and make the default prop value "grey100"